web/src/scripts/index.js

```diff
@@ -270,6 +342,106 @@ function getNextAnnouncement(id, tries) {
     });
 }
 
+// carousel management
+const itemClassName = 'carousel-entry';
+const items = document.getElementsByClassName(itemClassName);
+const totalItems = 3;
+var slide = 0;
+var moving = true;
+
+function setInitialClassesForCarousel() {
+	items[totalItems-1].classList.add('prev');
```

Comment:
On pages without a carousel this throws an errors because items.length === 0 and because js can be weird this stops all js execution on the page so on the apply page (which doesn't have a carousel) no js executes.